src/server/scripts/Spells/spell_warlock.cpp

```diff
@@ -109,6 +109,27 @@ class spell_warl_banish : public SpellScriptLoader
         }
 };
 
+// 116858 - Chaos Bolt
+class spell_warl_chaos_bolt : public SpellScript
+{
+    PrepareSpellScript(spell_warl_chaos_bolt);
+
+    bool Load() override
+    {
+        return GetCaster()->GetTypeId() == TYPEID_PLAYER;
+    }
+
+    void HandleDummy(SpellEffIndex /*effIndex*/)
+    {
+        SetHitDamage((GetHitDamage() * 2) + CalculatePct(GetHitDamage(), GetCaster()->ToPlayer()->m_activePlayerData->SpellCritPercentage));
```

Comment:
`damage * 2` is wrong - thats only a tooltip value (`dealing a critical strike for ${2*$s1} Chaos damage`), it includes the crit multiplier (but base damage shouldn't)

Missing `OnCalcCritChance` hook to always force critical strike

Reply:
oh i forgot about the critchance, because blizzard its changing descriptions always xd